docs/citing.rst

```diff
@@ -6,35 +6,159 @@
+  <span class="SDC_text_TOPUP" style="display: none">Distortion correction was performed using an implementation of the TOPUP technique [9] using 3dQwarp v16.2.07 distributed as part of AFNI [10]. </span>
+  <span class="SDC_text_FUGUE" style="display: none">Distortion correction was performed using fieldmaps processed by the FUGUE v5.0.9 [11] tool. </span>
+  <span class="SDC_text_SyN" style="display: none">Distortion correction was performed by estimating ANTs based nonlinear coregistration of the functional image to intensity inverted T1w image [12,13] constrained with an average fieldmap template [14].</span>
+  This was followed by coregistration to the corresponding T1 weighted volume using boundary based registration 9 degrees of freedom - implemented in
+  <span class="freesurfer_text_true">FreeSurfer v6.0.0 [15]</span>
+  <span class="freesurfer_text_false" style="display: none">FSL [15]</span>
+  . Motion correcting transformations,
+  <span class="SDC_text_TOPUP" style="display: none">field distortion correcting warp, </span>
+  <span class="SDC_text_FUGUE" style="display: none">field distortion correcting warp, </span>
+  <span class="SDC_text_SyN" style="display: none">field distortion correcting warp, </span>
+  T1 weighted transformation and MNI template warp were applied in a single step using antsApplyTransformations v2.1.0 with Lanczos interpolation.
+  </p>
+  <p style="font-style: italic;">
+  Three tissue classes were extracted from T1w images using FSL FAST v5.0.9 [16]. Voxels from cerebrospinal fluid and white matter were used to create a mask in turn used to extract physiological noise regressors using aCompCor [17]. Mask was eroded and limited to subcortical regions to limit overlap with grey matter, six principal components were estimated. Framewise displacement [18] was calculated for each functional run using Nipype implementation. <span class="AROMA_text_true" style="display: none">ICA-based Automatic Removal Of Motion Artifacts (AROMA) was used to generate aggressive noise regressors as well as creating a variant of data that was non-aggressively denoised [19].</span></p>
+  <p style="font-style: italic;">
+  For more details of the pipeline see <a id="workflow_url" href="http://fmriprep.readthedocs.io/en/latest/workflows.html">http://fmriprep.readthedocs.io/en/latest/workflows.html</a>.
+  </p>
+  <p>
+  <span id="fmriprep_citation">FMRIPREP</span> Available from: <a id="fmriprep_doi_url" href="https://doi.org/10.5281/zenodo.852659">https://doi.org/10.5281/zenodo.852659</a>
+  <img src onerror='fillCitation()'>
+  </p>
+  <p>
+  2. 	Gorgolewski K, Burns CD, Madison C, Clark D, Halchenko YO, Waskom ML, Ghosh SS. Nipype: a flexible, lightweight and extensible neuroimaging data processing framework in python. Front Neuroinform [Internet]. 2011 Aug 22;5(August):13. Available from: <a href="http://doi.org/10.3389/fninf.2011.00013">http://doi.org/10.3389/fninf.2011.00013</a>
```

Comment:
Sorry, wasn't clear. The main thing I was suggesting with the DOI URLs was the **s** in `https`, more than just dropping the `dx`. (I suspect https://dx.doi.org works, but figure shorter URLs are better.)